flink/flink-common/src/main/java/com/netease/arctic/flink/catalog/ArcticCatalog.java

```diff
@@ -289,29 +287,31 @@ public void renameTable(ObjectPath tablePath, String newTableName, boolean ignor
   public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ignoreIfExists)
       throws CatalogException, TableAlreadyExistException {
     validateFlinkTable(table);
+    createArcticTable(tablePath, table, ignoreIfExists);
+  }
 
+  private void createArcticTable(
+      ObjectPath tablePath, CatalogBaseTable table, boolean ignoreIfExists)
+      throws CatalogException, TableAlreadyExistException {
     TableSchema tableSchema = table.getSchema();
     TableSchema.Builder flinkSchemaBuilder = TableSchema.builder();
-
+    // add PhysicalColumn to arcticTableSchema
```

Comment:
Maybe we can generate the physical schema with `com.netease.arctic.flink.FlinkSchemaUtil#getPhysicalSchema`.